packages/sdk/src/components/Notebook/NotebookCell.tsx

```diff
@@ -129,16 +127,15 @@ export function NotebookCell({
                   }}
                >
                   <Typography
+                     component="div"
                      sx={{
                         fontSize: "12px",
                         "& .line": { textWrap: "wrap" },
                      }}
+                     dangerouslySetInnerHTML={{
```

Comment:
Unrelated change, but fixes a react warning about invalid DOM hierarchy (same for Typography change below as well)